Sources/TensorFlow/Operators/Math.swift

```diff
@@ -408,6 +408,20 @@ internal func _vjpLog<T: TensorFlowFloatingPoint>(
     return (log(x), { v in v / x })
 }
 
+/// Computes the base-two logarithm of the specified tensor element-wise.
+@inlinable
+@differentiable(where T: TensorFlowFloatingPoint)
+public func log2<T: FloatingPoint>(_ x: Tensor<T>) -> Tensor<T> {
+    log(x) / log(Tensor(2))
```

Comment:
Does this work?

```suggestion
    log(x) / log(2)
```

Reply:
It doesn't work. I believe it's because `FloatingPoint` does not conform to `ExpressibleByFloatLiteral`.
```
/Users/danielzheng/swift-apis/Sources/TensorFlow/Operators/Math.swift:415:12: error: binary operator '/' cannot be applied to operands of type 'Tensor<T>' and '_'
    log(x) / log(2)
    ~~~~~~ ^ ~~~~~~
```

Reply:
`FloatingPoint` types do have an initializer that takes a literal. You can do `log(Scalar(2))`. 

That said, I think we should change all of these operators to require `Scalar: TensorFlowFloatingPoint` for consistency with other APIs.

Reply:
`log(T(2))` doesn't work because `FloatingPoint` (nor `TensorFlowFloatingPoint`) refine `ElementaryFunctions`.

I'll keep the code as is: `log(Tensor(2))`.